oaknational/Oak-Web-Application
# Lesq 883   remove share and download from sitemap - LESQ-883

## Description

Music year: 1971

-  Removes share and downloads urls from teachers sitemap
- Adds specialist lessons, units and subjects to sitemap
- Refactors to use browse and specialist MV for sitemap generation

Tech debt - move specialist schema to shared schema repo - https://www.notion.so/oaknationalacademy/Move-specialist-schema-to-oak-curriculum-schema-a944f3d21d0b46158a495e43959b2486


## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2550--oak-web-application.netlify.thenational.academy

There should be no share and downloads lesson links
There should be specialist links

https://deploy-preview-2550--oak-web-application.netlify.thenational.academy/sitemap.xml
https://deploy-preview-2550--oak-web-application.netlify.thenational.academy/teachers/sitemap.xml

Specialist starts here - https://deploy-preview-2550--oak-web-application.netlify.thenational.academy/teachers/sitemap-1.xml#:~:text=%3Cloc%3Ehttps%3A//www.thenational.academy/teachers/specialist/subjects%3C/loc%3E



## Checklist
